lib/assets/SnapshotAlertManager.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  SnapshotAlertManager.swift
+//
+//  Created by Maxime Britto on 06/01/2016.
+//  Copyright © 2016 dev2a. All rights reserved.
+//
+
+import Foundation
+import XCTest
+
+var alertInterruptionMonitor:NSObjectProtocol?
+
+func automaticallyDismissAlerts(testCase:XCTestCase) -> NSObjectProtocol
+{
+    return SnapshotAlertManager.automaticallyDismissAlerts(testCase)
+}
+
+func stopDismissingAlerts(testCase:XCTestCase) {
+    SnapshotAlertManager.stopDismissingAlerts(testCase)
+}
+
+func waitForAlertsToBeDismissed() {
+    SnapshotAlertManager.waitForAlertsToBeDismissed()
+}
+
```

Comment:
I followed the same format as the current SnapshotHelper.swift file. This allows the methods to be called without prefixing the call with SnapshotAlertManager.
But you are right about the global variable though, I changed it to a type property, I just have to figure out how to update the PR (I've always used subversion and I'm still discovering git)